en-US/snippets/2017/nov2017_mobile.lang

```diff
@@ -0,0 +1,255 @@
+## NOTE: Reference bug 1410997 and screenshots. Deadline: 10 November, 2017.
+## NOTE: Mobile contextual hints: character limitation includes spaces.
+## NOTE: Brand names stay in English except when noted.
+
+
+## MAX_LENGTH: 79
+# Message limit: 79 characters
+;Private browsing and nothing else.
+Private browsing and nothing else.
+
+
+## MAX_LENGTH: 22
+# Button label: 22 characters
+# Firefox Focus remains in English. Firefox Klar in German.
+;Get Firefox Focus
+Get Firefox Focus
+
+
+## MAX_LENGTH: 79
+# Message limit: 79 characters
+;Need always-on private browsing?
+Need always-on private browsing?
+
+
+## MAX_LENGTH: 22
+# Button label: 22 characters
+# Firefox Focus remains in English. Firefox Klar in German.
+;Try Firefox Focus
+Try Firefox Focus
+
+
+## MAX_LENGTH: 79
+# Message limit: 79 characters
+;Want a browser that's always private?
```

Comment:
Ideally we should use curly quotes, even if they don't. It's on you to decide if you want to fix them or not ;-)

Reply:
This one is fixed, i don't know why this window doesn't go away, :).